tables/TaQL/TaQLNodeHandler.cc

```diff
@@ -377,9 +431,22 @@ namespace casacore { //# NAMESPACE CASACORE - BEGIN
     return TaQLNodeResult();
   }
 
-  TaQLNodeResult TaQLNodeHandler::visitJoinNode (const TaQLJoinNodeRep&)
+  TaQLNodeResult TaQLNodeHandler::visitJoinNode (const TaQLJoinNodeRep& node)
   {
-    throw TableInvExpr("Joins are not implemented yet");
+    //# Add a TableParseJoin object.
```

Comment:
```suggestion
    // Add a TableParseJoin object.
```
Mix of python and C comment? ;)

Reply:
No; it stems from the .h file where a //# comment is ignored for the doxygen documentation.